src/commands/request/request.ts

```diff
@@ -107,15 +107,17 @@ export async function runSasJob(
         return
       }
 
-      if (res?.result) res = res.result
-      await writeOutput(outputPathParam, res, isLocal)
+      let output = ''
+      if (res?.result) output = res.result
+      await writeOutput(outputPathParam, output, isLocal)
+      await saveLogFile(sasjs, sasJobLocation, logFile, jobPath)
       result = true
     })
     .catch(async (err: any) => {
       if (err && err.errorCode === 404) {
         displaySasjsRunnerError(configJson.username)
       } else {
-        displayError(err, 'An error occurred while executing the request.')
+        displayError('', 'An error occurred while executing the request.')
```

Comment:
what is the reason for not displaying `err` ?

Reply:
If there is an `err.errorCode` that would be quite relevant.  The problem is that SAS can fail for many reasons, so without those details, it becomes impossible for the end user to debug